code/modules/antagonists/abductor/machinery/experiment.dm

```diff
@@ -116,52 +116,51 @@
  * * type The type of experiment to be performed
  * * user The mob starting the experiment
  */
-/obj/machinery/abductor/experiment/proc/experiment(mob/occupant, type, mob/user)
+/obj/machinery/abductor/experiment/proc/experiment(mob/living/carbon/human/occupant, type, mob/user)
 	LAZYINITLIST(history)
-	var/mob/living/carbon/human/H = occupant
 
-	if(istype(H)) //We shouldn't be processing anything other than humans, and if we do we runtime.
```

Comment:
i feel like we should still have a `stack_trace()` since runtiming might have been intended to occur to root out bugs.